lib/OpenQA/Setup.pm

```diff
@@ -148,6 +149,7 @@ sub read_config ($app) {
             screenshot_cleanup_batch_size => OpenQA::Task::Job::Limit::DEFAULT_SCREENSHOTS_PER_BATCH,
             screenshot_cleanup_batches_per_minion_job => OpenQA::Task::Job::Limit::DEFAULT_BATCHES_PER_MINION_JOB,
             results_min_free_disk_space_percentage => undef,
+            expire_minion_jobs => ONE_WEEK,
```

Comment:
For consistency this should start with a noun.
```suggestion
            minion_job_max_age => ONE_WEEK,
```

Reply:
done